### PR TITLE
Update IconMgr.cpp

### PR DESCRIPTION
**Pull request by Pilot-Pirx**
Fixed typo

### CODE REVIEW

File: src/IconMgr.cpp [hunk 1]
```diff
@@ -13,7 +13,7 @@ void WiconMgr::grabaIcono(char* fileName){
 
 Os2IconMgr12* WiconMgr::codecOs2Icon12(){
    Os2IconMgr12 *iconoWarp = new Os2IconMgr12();
-   //Se parte de la imagen mas peque�a para que el WPS asuma la imagen mas grande al desplegarla
+   //Se parte de la imagen mas peque¤a para que el WPS asuma la imagen mas grande al desplegarla
```
Comment (komh) [inline src/IconMgr.cpp:16]: What's the difference?

File: src/IconMgr.cpp [hunk 2]
```diff
@@ -70,7 +70,7 @@ void WiconMgr::cargaIcono(char* fileName) throw (IconError){
    else{
       rc = DosQueryFileInfo(handleArchivo,FIL_STANDARD,&fileInfo,bufSize);
       if(rc != 0){
-         cout<<"Error al traer la informaci�n del archivo" <<endl;
+         cout<<"Error al traer la informaci¢n del archivo" <<endl;
```
Comment (komh) [inline src/IconMgr.cpp:73]: Here, too.

File: src/IconMgr.cpp [hunk 3]
```diff
@@ -308,7 +308,7 @@ Os2Icon12* Os2IconMgr12::creaGrande(Os2Icon12 *warpIcon){
    *pArrayAux = *granWarpIcon->getPArray();
    //Si se trata de una imagen de 16x16 seteo los encabezados de las
    //estructuras con los de una imagen de 20x20 y configuro los datos
-   //para los nuevos tama�os
+   //para los nuevos tama¤os
```
Comment (komh) [inline src/IconMgr.cpp:311]: Here, too.

File: src/IconMgr.cpp (hunk 1 above)
Comment (Pilot-Pirx) [reply src/IconMgr.cpp:16]: GitHub changed the encoding to UTF. No optical difference for me.

File: src/IconMgr.cpp (hunk 2 above)
Comment (Pilot-Pirx) [reply src/IconMgr.cpp:73]: See above.

File: src/IconMgr.cpp (hunk 3 above)
Comment (Pilot-Pirx) [reply src/IconMgr.cpp:311]: See above.

File: src/IconMgr.cpp (hunk 1 above)
Comment (komh) [reply src/IconMgr.cpp:16]: Then, remove these differences, please...

Comment (Pilot-Pirx) [reply src/IconMgr.cpp:16]: GitHub changed the encoding. Nothing I can do about it...

Feel free to close this PR. Maybe you can correct the single typo in line 28.

Comment (komh) [reply src/IconMgr.cpp:16]: I suspect CRLF vs LF problem. Anyway I'll  fix a typo and close this PR. 

Thanks!

**Pull request closed without merging**